app/scenes/Settings/Details.js

```diff
@@ -14,137 +13,130 @@ import Input, { LabelText } from "components/Input";
-              <Avatar src={avatarUrl} />
-              <Flex auto align="center" justify="center">
-                Upload
-              </Flex>
-            </ImageUpload>
-          </AvatarContainer>
-        </ProfilePicture>
-        <form onSubmit={this.handleSubmit} ref={(ref) => (this.form = ref)}>
-          <Input
-            label="Name"
-            name="name"
-            autoComplete="organization"
-            value={this.name}
-            onChange={this.handleNameChange}
-            required
-            short
-          />
-          {env.SUBDOMAINS_ENABLED && (
-            <>
-              <Input
-                label="Subdomain"
-                name="subdomain"
-                value={this.subdomain || ""}
-                onChange={this.handleSubdomainChange}
-                autoComplete="off"
-                minLength={4}
-                maxLength={32}
-                short
-              />
-              {this.subdomain && (
-                <HelpText small>
-                  Your knowledge base will be accessible at{" "}
-                  <strong>{this.subdomain}.getoutline.com</strong>
-                </HelpText>
-              )}
-            </>
-          )}
-          <Button type="submit" disabled={isSaving || !this.isValid}>
-            {isSaving ? "Saving…" : "Save"}
-          </Button>
-        </form>
-      </Scene>
-    );
-  }
+        </Trans>
+      </HelpText>
+
+      <ProfilePicture column>
+        <LabelText>{t("Logo")}</LabelText>
+        <AvatarContainer>
+          <ImageUpload
+            onSuccess={handleAvatarUpload}
+            onError={handleAvatarError}
+            submitText={t("Crop logo")}
+            borderRadius={0}
+          >
+            <Avatar src={avatarUrl} />
+            <Flex auto align="center" justify="center">
+              <Trans>Upload</Trans>
+            </Flex>
+          </ImageUpload>
+        </AvatarContainer>
+      </ProfilePicture>
+      <form onSubmit={handleSubmit} ref={form}>
+        <Input
+          label={t("Name")}
+          name="name"
+          autoComplete="organization"
+          value={name}
+          onChange={handleNameChange}
+          required
+          short
+        />
+        {env.SUBDOMAINS_ENABLED && (
+          <>
+            <Input
+              label={t("Subdomain")}
+              name="subdomain"
+              value={subdomain || ""}
+              onChange={handleSubdomainChange}
+              autoComplete="off"
+              minLength={4}
+              maxLength={32}
+              short
+            />
+            {subdomain && (
+              <HelpText small>
+                <Trans>Your knowledge base will be accessible at</Trans>{" "}
+                <strong>{subdomain}.getoutline.com</strong>
+              </HelpText>
+            )}
+          </>
+        )}
+        <Button type="submit" disabled={auth.isSaving || !isValid}>
+          {auth.isSaving ? t("Saving…") : t("Save")}
```

Comment:
```suggestion
          {auth.isSaving ? `${t("Saving")}…` : t("Save")}
```

It's a pita from a syntax pov but it's better for the translations